src/catERC20Factory.sol

```diff
@@ -3,8 +3,10 @@ pragma solidity >=0.8.4 <0.9.0;
 
 import { IXERC20Factory } from './interfaces/IXERC20Factory.sol';
 
-import { CatERC20 } from './CatERC20.sol';
-import { CatLockbox } from './CatLockbox.sol';
+import { CatERC20 } from './catERC20.sol';
+import { CatLockbox } from './catLockbox.sol';
+
```

Comment:
There is no need to modify the catERC20Factory for this commit.
This is bad style as it makes it hardier to ensure that this PR does not touch core code.